challenge_2/python/tvanderwal/src/challenge_2.py

```diff
@@ -0,0 +1,7 @@
+from collections import Counter
+#extra credit list
+# Added 5 to the end so the expected output in instructions would be correct
+myList = (2,'a','l',3,'l',4,'k',2,3,4,'a',6,'c',4,'m',6,'m','k',9,10,9,8,7,8,10,7,5)
+
+#use counter to see if item is repeated and a list comprehension to store them
+print([k for k, v in Counter(myList).items() if v == 1])
```

Comment:
I think that it's inefficient to have a loop call Counter each time, since Counter creates a dictionary (hash table) each time you call it. It might be better to first create a variable in which you store the Counter information, and then loop over that once, so you have a "looping instance" occur twice, rather than a looping instance occur within your `for` loop. I may be wrong, but I believe this is O(n^2) rather than O(n).

That said, it is correct! Very nicely styled.